models/balancer/balancer_gauge_mappings.sql

```diff
@@ -0,0 +1,37 @@
+{{ config(
+    schema = 'balancer',
+    alias = 'gauge_mappings',
+    post_hook='{{ expose_spells(\'["optimism"]\',
+                                "project",
+                                "balancer",
+                                \'["msilb7"]\') }}'
+    )
+}}
+
+{% set balancer_models = [
+    ref('balancer_optimism_gauge_mappings')
+] %}
+
+
+SELECT *
+FROM (
+    {% for gauge_model in balancer_models %}
+    SELECT
+        blockchain
+        , 'balancer' as project
+        , '2' as version
```

Comment:
you may find it easier to keep version derived in the base model, rather than here, in case you add in the future and versions differ

Reply:
fixed